java/form/src/org/netbeans/modules/form/JavaCodeGenerator.java

```diff
@@ -2554,6 +2519,84 @@ private void generateListenerAddCode(RADComponent comp,
             generateCatchCode(exceptions, codeWriter);
     }
 
+    private void generateInnerClasses(Writer codeWriter, EventSetDescriptor eventSetDesc, List<Event> eventList, boolean useLambdas) throws IOException {
+
+        if (useLambdas && eventSetDesc.getListenerMethods().length == 1) {
+            generateWithReferenceOrLambda(codeWriter, eventList);
+        } else {
+            codeWriter.write("new "); // NOI18N
```

Comment:
I looked over this and confirmed that the code pasted here in the "else" case is the same as the above, with some whitespace changes only.